library/src/main/java/com/getstream/sdk/chat/adapter/MessageViewHolderFactory.kt

```diff
@@ -37,9 +36,9 @@ open class MessageViewHolderFactory {
     }
 
     open fun getAttachmentViewType(
-        attachment: Attachment
+        attachmentItem: AttachmentListItem
     ): Int {
-        return when (attachment.type) {
+        return when (attachmentItem.attachment.type) {
```

Comment:
It seems that `AttachmentListItem` could be a `sealed class`, so it will be easier to handle multiple types.
It could be covered on another PR when `ViewHolder` and `Adapter` will refactored

Reply:
I'd leave this for later, the wrapper feels like enough preparation for future expansion for now